ui/Screen/Bsod.svelte

```diff
@@ -14,29 +21,71 @@
   }
 
   .container {
+    height: 100%;
+    padding: 5rem;
+
     display: flex;
-    text-align: center;
     justify-content: center;
     align-items: center;
     flex-direction: column;
     color: #fff; /* I know... but this design doesn't work in dark mode. */
   }
+  .proxy-log-container {
+    background: #e3e3ff;
+
+    max-width: 100%;
+    min-width: 40em;
+    max-height: 20vh;
+    overflow: scroll;
+
+    margin-top: 2rem;
+    border-radius: 8px;
+    padding: 0.9em;
+  }
+
+  .proxy-log {
+    display: block;
+    color: #5555ff;
+    font-size: 14px;
+    white-space: pre-wrap;
+    line-height: 1.4;
+
+    max-width: 100%;
+  }
 </style>
 
-{#if $fatalError}
+{#if $fatalError !== null}
   <Fullscreen
     escapable={false}
-    style="background-color: var(--color-secondary); z-index: 200;"
-    contentStyle="display: flex; width: 312px; align-items: center;">
+    style="background-color: var(--color-secondary); z-index: 200;">
     <div class="container" data-cy="blue-screen-of-death">
       <Emoji emoji="🧻" size="huge" style="margin-bottom: 1.5rem;" />
-      <p>
-        We're not totally sure what's going on, but we can't load the app right
-        now because the background process won't start up.
+      <p style="width: 321px; text-align: center">
+        {#if $fatalError.kind === 'SESSION'}
+          We're not totally sure what's going on, but we can't load the app
+        {:else if $fatalError.kind === 'PROXY_EXIT'}
+          Hmm, looks like the app can’t be loaded right now because the backend
+          has crashed or it isn’t starting.
+        {/if}
       </p>
       <Button style="display: flex; color: #fff;" on:click={tweet}>
         Reach out for support
       </Button>
+      {#if $fatalError.kind === 'PROXY_EXIT' && $fatalError.data.stderr}
+        <div class="proxy-log-container">
+          <code data-cy="proxy-log" class="proxy-log typo-mono-semi-bold">
+            {$fatalError.data.stderr}
+          </code>
+          <Button
+            dataCy="proxy-log-copy-clipboard"
+            style="position: sticky; bottom: 0; margin-left: auto;"
+            variant="secondary"
+            on:click={() => ipc.copyToClipboard($fatalError.data.stderr)}
```

Comment:
Could we keep the clipboard error format consistent across the app?

For error notifications we already use:
```
JSON.stringify({ message, context }, null, 2)
```

`message` in this case would either be:
  - "We're not totally sure what's going on, but we can't load the app" or 
  - "Hmm, looks like the app can’t be loaded right now because the backend has crashed or it isn’t starting." 

depending on `$fatalError.kind`.

`context` -> `$fatalError.data.stderr`

https://github.com/radicle-dev/radicle-upstream/blob/master/ui/src/error.ts#L20

Reply:
One significant drawback of this would be that the clipboard content cannot be easily read without first extracting the log again. For example it would look like this
~~~json
{
  "context":" INFO  coco::git_helper > Copied git remote helper to: \"/home/thomas/.radicle/bin/git-remote-rad\"\n INFO  api::process     > starting API\n ERROR api::http::error > Rejection(KeystoreSealed)\n INFO  proxy::http      > \"GET /v1/session HTTP/1.1\" 403 1.38934ms\n INFO  proxy::http      > \"POST /v1/keystore/unseal HTTP/1.1\" 204 12.649689597s\n INFO  librad::net::protocol > Protocol::run; local.id=hybutqgn3dnorg5gcsk3unj841b5bgkihmgoaagq7em8xdngkw71ey local.addr=0.0.0.0:43752\n INFO  api::process          > starting peer\n INFO  api::process          > starting API\n INFO  librad::net::protocol > Listening \n INFO  librad::net::protocol > Establishing connection remote.id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw \n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  librad::net::protocol > New outgoing connection remote.id=hynewpywqj6x4mxgj7sojhue3erucyexiyhobxx4du9w66hxhbfqbw \n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"GET /v1/session HTTP/1.1\" 200 1.782777ms\n INFO  proxy::http           > \"GET /v1/notifications/local_peer_events HTTP/1.1\" 200 1.430587ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"GET /v1/projects/failed HTTP/1.1\" 200 4.503578ms\n INFO  proxy::http           > \"GET /v1/projects/contributed HTTP/1.1\" 200 4.849867ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"GET /v1/projects/requests HTTP/1.1\" 200 1.627793ms\n INFO  proxy::http           > \"GET /v1/projects/tracked HTTP/1.1\" 200 4.705862ms\n INFO  proxy::http           > \"GET /v1/projects/requests HTTP/1.1\" 200 2.449423ms\n INFO  proxy::http           > \"GET /v1/projects/tracked HTTP/1.1\" 200 4.877787ms\n INFO  proxy::http           > \"GET /v1/projects/requests HTTP/1.1\" 200 1.691161ms\n INFO  proxy::http           > \"GET /v1/projects/tracked HTTP/1.1\" 200 3.971539ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"GET /v1/projects/failed HTTP/1.1\" 200 11.838687ms\n INFO  proxy::http           > \"GET /v1/projects/contributed HTTP/1.1\" 200 12.56111ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"POST /v1/session/settings HTTP/1.1\" 204 1.697896ms\n INFO  proxy::http           > \"GET /v1/session HTTP/1.1\" 200 1.338659ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"POST /v1/session/settings HTTP/1.1\" 204 4.735288ms\n INFO  proxy::http           > \"GET /v1/session HTTP/1.1\" 200 2.440608ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  proxy::http           > \"GET /v1/projects/failed HTTP/1.1\" 200 4.894313ms\n INFO  proxy::http           > \"GET /v1/projects/contributed HTTP/1.1\" 200 4.994706ms\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  librad::net::protocol > Establishing connection remote.id=hydzh4g7aue8o44foafamatq6jbwiza64xbdudkuy9arucrieyymey \n INFO  quinn::connection     > drive; id=1\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=1\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=1\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n INFO  quinn::connection     > drive; id=0\n"
}
~~~

I also don’t think we gain anything by enforcing this structure rigidly on the the clipboard message. The message is sent to us by individuals and we will not process them automatically. For most other errors this makes sense though.

Reply:
Hm, you're right, it would be harder to read. Never mind then.